products/cakephp.md

```diff
@@ -6,9 +6,8 @@ permalink: /cakephp
 alternate_urls:
 -   /cake-php
 -   /cake
-releasePolicyLink: https://book.cakephp.org/4/en/release-policy.html
+releasePolicyLink: https://github.com/cakephp/cakephp/security/policy#supported-versions
 changelogTemplate: https://github.com/cakephp/cakephp/releases/__LATEST__
-releaseLabel: "__RELEASE_CYCLE__ __CODENAME__"
```

Comment:
Major releases have code names.

CakePHP 3.x was "Red Velvet"

> New CakePHP 3.3 Red Velvet.

https://web.archive.org/web/20170105051444/https://cakephp.org/

> New CakePHP 3.7 Red Velvet.

https://web.archive.org/web/20190206051719/https://cakephp.org/

CakePHP 4.x was "Strawberry"

> New CakePHP 4.1 Strawberry.

https://web.archive.org/web/20210207001445/https://cakephp.org/

> New CakePHP 4.3 Strawberry.

https://web.archive.org/web/20220215212220/https://cakephp.org/

CakePHP 5 will be "Chiffon".

https://www.facebook.com/CakePHP/posts/671569178092040

I'm open to handle this differently.

Reply:
We have 3 choices :

1. code name on all 3.x and 4.x versions (and 2.x / 1.x if they have a code name too),
2. code name on the last 3.x and 4.x,
3. no code name at all.

It would be strange to have the same code name for all versions in the same major branch. And it's also strange to have a code name only for the last version in the cycle (because users will think this is the code name for this particular minor version). So I am more enclined to remove it.

Note that it would not be a problem if only major versions were tracked ;).

Reply:
Closing this conversation, discussion moved to https://github.com/endoflife-date/endoflife.date/pull/2290.